packages/orm/src/Aliases.ts

```diff
@@ -27,13 +27,15 @@ export function aliases<T extends readonly MaybeAbstractEntityConstructor<any>[]
 }
 
 export type Alias<T extends Entity> = {
-  [P in keyof FieldsOf<T>]: FieldsOf<T>[P] extends { kind: "primitive" | "enum"; type: infer V; nullable: infer N }
-    ? PrimitiveAlias<V, N extends undefined ? null : never>
-    : FieldsOf<T>[P] extends { kind: "m2o"; type: infer U }
-      ? EntityAlias<U>
-      : FieldsOf<T>[P] extends { kind: "poly"; type: infer U extends Entity }
-        ? PolyReferenceAlias<U>
-        : never;
+  [P in keyof FieldsOf<T>]: P extends "id"
```

Comment:
Put a special case here for `"id`" to get it flipped from a `PrimiviteAlias<string>` to `EntityAlias<T>`, which seems more accurate.